slack-lint/src/test/java/slack/lint/DeprecatedAnnotationDetectorTest.kt

```diff
@@ -193,7 +193,7 @@ class DeprecatedAnnotationDetectorTest : BaseSlackLintTest() {
       .run()
       .expect(
         """
-          src/slack/test/TestClass.kt:8: Warning: This class or method is deprecated; consider using an alternative. [DeprecatedCall]
+          src/slack/test/TestClass.kt:8: Warning: slack.test.ThisIsDeprecated.ThisIsDeprecated is deprecated; consider using an alternative. [DeprecatedCall]
```

Comment:
Kind of weird, but this is basically pointing at the constructor method. Fine IMO